assets/js/components/DottedPagination/Arrow.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 
-function Arrow({ children, onClick }) {
+function Arrow({ children, onClick, datatestID }) {
```

Comment:
You can use `...rest` here, spreading the `...rest` onto the `div` returned by the element, and pass `data-testid` from the outside without worrying about the casing

Reply:
Nice! I didn't think about it and it looks way better!